joomla/joomla-cms
# UX Isis: Improve sidebar and style fixes

- Added height to 100% (sidebar)
- Adding background to sidebar
- Fixed logo spacing
- Form style fixed
#### Sidebar

![sidebarheight](https://cloud.githubusercontent.com/assets/876623/6657245/e981d662-cb45-11e4-9e3b-5c6c145f4225.jpg)
#### Components Options

![sidebar](https://cloud.githubusercontent.com/assets/876623/6657255/0e7ec27c-cb46-11e4-9bd3-107aaf189b70.jpg)
#### Global Settings

![global](https://cloud.githubusercontent.com/assets/876623/6657258/1af5a7dc-cb46-11e4-981d-2a5217982463.jpg)
#### How to test

Apply patch. 
- See that the logo on the right is correct height.
- See that yes/no options are the same width as dropdowns on the config, also check other items on settings.
- See the improvements on Global and Options.
